pypesto/visualize/profiles.py

```diff
@@ -67,21 +71,27 @@ def profiles(results, ax=None, profile_indices=None, size=(18.5, 6.5),
         fvals = handle_inputs(result, profile_indices=profile_indices,
                               profile_list_id=profile_list_id)
 
+        if x_labels is None:
+            x_labels = [f'Value of {name}' for name, fval in
+                        zip(result.problem.x_names, fvals) if fval is not None]
```

Comment:
I would probably go for
```suggestion
            x_labels = [name for name, fval in
                        zip(result.problem.x_names, fvals) if fval is not None]
```